src/scripts/gen_rewards_dist.js

```diff
@@ -13,8 +13,8 @@ const MerkleDist = require("../merkle_dist/merkle_dist.js")
 const bonusWeight = 0.0
 const preWeight = 0.5
 const tbtcv2Weight = 0.5
-const startTime = 1672531200 // Jan 1st 2023 00:00:00 GMT
-const endTime = 1675209600 // Feb 1st 2023 00:00:00 GMT
+const startTime = new Date("2023-01-01T00:00:00+00:00").getTime() / 1000
+const endTime = new Date("2023-02-01T00:00:00+00:00").getTime() / 1000
 const lastDistribution = "2023-01-01"
```

Comment:
Does this value need updating to "2023-02-01"?

Reply:
No, not really. `lastDistribution` is used to add the earnings of the new distribution to the accumulated rewards of the last distribution. At the moment of generating the February distribution, the last distribution was the January one.

Reply:
Ah ok. Makes sense. Thanks for clarifying. 👌